templates/vue/src/components/modals/NodeModal/forms/ContentForm/ChoiceRow.vue

```diff
@@ -0,0 +1,154 @@
+      <b-form-input
+        v-model="item.value"
+        class="formInput"
+        :placeholder="placeholder"
+        :data-qa="`choicerow-checkbox-input-${item.id}`"
+      ></b-form-input>
+    </div>
+    <b-button
+      :disabled="removeButtonDisabled"
+      squared
+      variant="outline-danger"
+      @click="$emit('remove')"
+    >
+      Remove
+    </b-button>
+  </div>
+
+  <div v-else class="container">
+    <span id="handle" v-handle class="fas fa-bars fa-s"></span>
+    <b-form-checkbox
+      :value="item.id"
+      :disabled="isDisabled && selectedRadioChoice != item.id"
+    >
+      <b-form-group v-if="useImages">
+        <file-upload
+          v-model="item.imageurl"
+          input-test-id="node-choiceRow-thumbnail-url"
+          :data-qa="`choicerow-radio-thumbnail-${item.id}`"
+          :show-url-upload="false"
+          :show-image-preview="true"
+          file-types="image/*"
+          @isUploading="handleUploadChange"
+        />
+      </b-form-group>
+    </b-form-checkbox>
+    <div :class="useImages ? 'inputContainerImage' : 'inputContainer'">
+      <b-form-input
+        v-model="item.value"
+        class="formInput"
+        :placeholder="placeholder"
+        :data-qa="`choicerow-radio-input-${item.id}`"
+      ></b-form-input>
+    </div>
+    <b-button
+      :disabled="removeButtonDisabled"
+      squared
+      variant="outline-danger"
+      @click="$emit('remove')"
+    >
+      Remove
+    </b-button>
+  </div>
+</template>
+
+<script>
+import FileUpload from "@/components/modals/common/FileUpload"
+import { HandleDirective, ElementMixin } from "vue-slicksort"
+export default {
+  components: {
+    FileUpload,
+  },
+  directives: { handle: HandleDirective },
+  mixins: [ElementMixin],
+  props: {
+    question: {
+      type: Object,
+      required: true,
+    },
+    multipleAnswerSelected: {
+      type: Boolean,
+      required: true,
+    },
+    index: {
+      type: Number,
+      required: true,
+    },
+    placeholder: {
+      type: String,
+      required: false,
+      default: "",
+    },
+    useImages: {
+      type: Boolean,
+      required: true,
+    },
+    item: {
+      type: Object,
+      required: true,
+    },
+    isDisabled: {
+      type: Boolean,
+      required: false,
+      default: false,
+    },
+    selectedRadioChoice: {
+      type: Number,
+      required: false,
+      default: 0,
+    },
+    removeButtonDisabled: {
+      type: Boolean,
+      required: false,
+      default: false,
+    },
+  },
+  methods: {
+    handleUploadChange(state) {
+      this.$root.$emit("node-modal::uploading", state)
+    },
+  },
+}
+</script>
+
+<style lang="scss">
+.inputContainer {
+  margin-right: -20px;
+  width: 500px;
+}
+.inputContainerImage {
+  margin-right: -20px;
+  width: 100px;
+}
+.formInput {
+  margin-left: -20px;
+}
+.inputchoice {
+  width: 50%;
+}
+.container {
+  display: flex;
+  align-items: center;
+  z-index: 9999 !important;
+}
+#handle {
```

Comment:
Avoid using IDs because IDs are supposed to be unique and since you have multiple choice rows, you would end up with non-unique handles. If you are using an ID, please make them unique.